script/int-log-compiler-ptpm.sh

```diff
@@ -92,6 +92,18 @@ if [ $? -ne 0 ]; then
     break
 fi
 
+TPMSTATE_FILE1=${TEST_BIN}_state1
+TPMSTATE_FILE2=${TEST_BIN}_state2
+
+env TPM20TEST_TCTI_NAME="device" \
+    TPM20TEST_DEVICE_FILE=${PTPM} \
+    G_MESSAGES_DEBUG=all ./test/helper/tpm_dumpstate>$TPMSTATE_FILE1
+if [ $? -ne 0 ]; then
```

Comment:
You need to add the test/integration/*.int_state[2|1] files to the $CLEANFILES otherwise make distcheck will complain.